src/breadcrumb-group/item/item.tsx

```diff
@@ -24,24 +122,32 @@ export function BreadcrumbItem<T extends BreadcrumbGroupProps.Item>({
     }
     fireCancelableEvent(onClick, getEventDetail(item), event);
   };
+
+  const anchorAttributes: React.AnchorHTMLAttributes<HTMLAnchorElement> = {
+    href: isLast ? undefined : item.href || '#',
+    className: clsx(styles.anchor, { [styles.compressed]: isCompressed }),
+    'aria-current': isLast ? 'page' : undefined, // Active breadcrumb item is implemented according to WAI-ARIA 1.1
+    'aria-disabled': isLast && 'true',
+    onClick: isLast ? preventDefault : onClickHandler,
+    tabIndex: isLast ? 0 : undefined, // tabIndex is added to the last crumb to keep it in the index without an href
+  };
+
   return (
-    <div className={clsx(styles.breadcrumb, isLast && styles.last)}>
-      <a
-        {...focusVisible}
-        href={isLast ? undefined : item.href || '#'}
-        className={clsx(styles.anchor, { [styles.compressed]: isCompressed })}
-        aria-current={isLast ? 'page' : undefined} // Active breadcrumb item is implemented according to WAI-ARIA 1.1
-        aria-disabled={isLast && 'true'}
-        onClick={isLast ? preventDefault : onClickHandler}
-        tabIndex={isLast ? 0 : undefined} // tabIndex is added to the last crumb to keep it in the index without an href
-      >
-        <span className={styles.text}>{item.text}</span>
-      </a>
-      {!isLast ? (
-        <span className={styles.icon}>
-          <InternalIcon name="angle-right" />
-        </span>
-      ) : null}
-    </div>
+    <>
+      <div className={clsx(styles.breadcrumb, isLast && styles.last)}>
+        {(isLast || isFirst) && isCompressed ? (
```

Comment:
Can we simplify this condition by checking mobile mode once and rendering completely different branches?